### ISSUE TITLE
How to train for new speaker and minimum required dataset required?

### ISSUE THREAD

**Issue by navinthenapster** [Kyubyong/dc_tts#31]
Can you please guide me on how to train for new speaker with 1 minutes sample as mentioned

**Comment by iampartho** [comment]
Did you resolve the issue , can you teach me how you resolved it? Thanks.

**Comment by vinamramunot-tech** [comment]
@navinthenapster @iampartho Can you provide steps on how to create your own custom dataset and then how to train that dataset? Not sure how to run the training as not sure about the format of the dataset.

**Comment by iampartho** [comment]
Well it has been pretty long since I did the project so my memory is not that sharp about what stepped I followed for my project. But here it goes:

For the creation of dataset I think I used an app named lyrebird ai , it basically records like some minutes of voice data (which sort of use all sort of pronunciation and usually used words) , so I created my dataset using that app. I was  not good and comfortable in tensorflow and codes were not well documented here so I shifted from here and used [this](https://github.com/r9y9/deepvoice3_pytorch) project for my work as it was well documented and easy to implement. But honestly my results were not great but anyway I hope this information helps you.
